goss-testing/scripts/check_bgp_neighbors_established.sh

```diff
@@ -49,20 +49,52 @@ function err_exit
     exit $rc
 }
 
-kubectl -n metallb-system get cm metallb -o jsonpath='{.data.config}' > $TMPFILE ||
-    err_exit 10 "ERROR: Command failed (rc=$?): kubectl -n metallb-system get cm metallb -o jsonpath='{.data.config}' > $TMPFILE"
```

Comment:
Now that TMPFILE is no longer being used, lines 32-39 above and any invocation of "cleanup" can be removed.

Reply:
removed TMPFILE and cleanup function